packages/core/admin/admin/custom.d.ts

```diff
@@ -0,0 +1,8 @@
+export {};
```

Comment:
nice, this  "custom.d.ts" should be used for augmenting types from other packages right?

Reply:
yes!